e2e/cypress/common/translations.ts

```diff
@@ -10,28 +10,15 @@ import { ProjectDTO } from '../../../webapp/src/service/response.types';
 import Chainable = Cypress.Chainable;
 
 export function getCellEditButton(content: string) {
-  return cy
-    .contains(content)
-    .xpath(
-      "./ancestor::*[@data-cy='translations-table-cell']//*[@data-cy='translations-cell-edit-button']"
-    )
-    .invoke('show');
+  return cy.contains(content);
```

Comment:
I don't think, this would work for all possible cell contents.

Reply:
Changed